keywording/text.xml

```diff
@@ -267,6 +291,38 @@ Vulnerability Treatment Policy</uri>
 
 </body>
 
+<subsection>
+<title>Stabilization rules</title>
+<body>
+
+<p>
+SPARC: You must have prior permission from the arch lead. Usually we expect
```

Comment:
Maybe we should verify whether this is up-to-date. With sparc going exp, and dakon being the only person to stabilize...

Reply:
same goes for `hppa` in that case

Reply:
Can we postpone this to a separate commit, and not conflate it with merging of sections?